src/Core/Managed/Net46/Extensibility/Implementation/RichPayloadEventSource.cs

```diff
@@ -259,6 +274,11 @@ public sealed class Keywords
             /// Keyword for session state.
             /// </summary>
             public const EventKeywords SessionState = (EventKeywords)0x100;
+
+            /// <summary>
+            /// Keyword for availability.
+            /// </summary>
+            public const EventKeywords Availability = (EventKeywords)0x120;
```

Comment:
Should be 0x200.
(Because 0x120 would be SessionState + Metrics and not a new value)